src/app/pages/swap/components/swap-details/swap-details.tsx

```diff
@@ -59,7 +60,9 @@ export function SwapDetails() {
       <SwapDetailLayout
         dataTestId={SwapSelectors.SwapDetailsProtocol}
         title="Powered by"
-        value={swapSubmissionData.protocol}
+        value={`${Array.from(new Set(swapSubmissionData.dexPath))
+          .map(x => capitalize(x.toLowerCase()))
+          .join(', ')} via ${swapSubmissionData.protocol}`}
```

Comment:
Ideally this component remains agnostic from any specific dex implementation. e.g. the `dexPath` property is specific to Bitflow right? 

Wonder if this logic value/formatting logic could be defined in `BitflowSwapContainer` and pass in via the context?

Reply:
Either way, would be nice to move into a function that lives outside the JSX